index.html

```diff
@@ -0,0 +1,219 @@
+<!DOCTYPE html>
+<html lang="en">
+
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+
+		<title>Anna Russett</title>
+	
+		<link rel="icon" type="image/x-icon" href="http://www. /favicon.ico">
```

Comment:
i say we make a tinier version ( ~42x42 px ) of the cartoon face icon, what do u think?